scripts/gha/integration_testing/test_validation.py

```diff
@@ -184,7 +191,12 @@ def summarize_test_results(tests, platform, summary_dir, file_name="summary.log"
   if errors:
     summary.append("\n%d TESTAPPS EXPERIENCED ERRORS:" % len(errors))
     for test, results in errors:
-      summary.append("%s:" % test.testapp_path)
+      summary.append("\n%s:" % test.testapp_path)
+      if test.testapp_path in success_testapp_paths:
+        summary.append("THIS IS A FLAKY TESTAPP")
```

Comment:
I like it better when the keyword, in this case "FLAKY" either begins or ends the line.  It makes it easy to spot.  

How about `"THIS TESTAPP IS FLAKY"` instead?